lib/state.go

```diff
@@ -45,6 +45,7 @@ type State struct {
 	Options Options
 
 	// Logger. Avoid using the global logger.
+	// TODO change to logrus.FieldLogger when there is time to fix all the tests
 	Logger *logrus.Logger
```

Comment:
I think this TODO is already done, right?

Reply:
Unfortunately - no because in a lot of places (in tests) it is used in ways that will require even more code changes ... And when I started doing those it turned out I will need to change way too much code, and as I explained in the original commit message, this is not as important (or at all) for what we currently use it :) 

The thing that I started fixing (and likely not the only problem) is https://github.com/loadimpact/k6/blob/251a88d46ed47d917df877f79909fb7d1134f674/js/modules/k6/http/request_test.go#L293-L294 
We can get the logger the same way it is done in  https://github.com/loadimpact/k6/blob/251a88d46ed47d917df877f79909fb7d1134f674/js/console_test.go#L243-L246 but I decided that doing that will add unnecessary amount of additional work, for no practical benefit really